transformer_engine/pytorch/attention.py

```diff
@@ -2821,6 +2844,16 @@ class MultiheadAttention(torch.nn.Module):
           The device on which the parameters of the model will allocated. It is the user's
           responsibility to ensure all parameters are moved to the GPU before running the
           forward pass.
+    qkv_format: str, default = `sbhd`
+            dimension format for `query_layer`, `key_layer` and `value_layer`,
+            {`sbhd`, `bshd`}. `s` stands for the sequence length, `b` batch size,
+            `h` the number of heads, `d` head size, and `t` the total number of sequences
+            in a batch, with `t = sum(s_i), for i = 0...b-1`. `sbhd` and `bshd` formats
+            are used for when sequences in a batch are of equal length or padded to
+            equal length, and the `thd` format is used for when sequences in a batch
+            have different lengths. Please note that these formats do not reflect how
```

Comment:
You might want to remove the `thd` related explanations since we don't support thd here anyway? Thanks.